tools/dds/dds-server/lrs-device-manager.h

```diff
@@ -0,0 +1,28 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+#pragma once
+#include <librealsense2/rs.hpp>  // Include RealSense Cross Platform API
+#include <unordered_map>
+
+namespace tools {
+
+// This class is in charge of handling a RS device: streaming, control..
+class lrs_device_manager
+{
+public:
+    
+    lrs_device_manager( rs2::device dev );
+    ~lrs_device_manager();
+    void start_stream( rs2::stream_profile sp, std::function< void( const std::string&, uint8_t*, int ) > cb );
```

Comment:
Not clear here what the function parameters are

Reply:
Will be added